population/create_glpi_computer_redfish.py

```diff
@@ -181,6 +185,28 @@ def main() -> None:
         sunbird_url = validate_url(args.sunbird_url)
     else:
         sunbird_url = None
+    if all(
```

Comment:
What if the user inputs a subset of the flags? They will hit the need to specify machine exception below, which I think should be slightly more clear. I.e. it should either list the missing flags and/or mention which are required for population.